### ISSUE TITLE
Slider doesn't load on Android Jellyfin app

### ISSUE THREAD

**Issue by lachlandcp** [lachlandcp/jellyfin-editors-choice-plugin#5]
The slider doesn't load on the Android Jellyfin app, despite loading fine on a mobile web browser, and despite the app essentially being a wrapper for the website. The custom JS script doesn't seem to get run at all, despite being hardcoded into index.html on the webserver.

This one is stumping me so if anyone has any insights, that would be helpful. It's very strange.

**Comment by qwerty12** [comment]
From my phone, in order:
```
script:125 Uncaught ReferenceError: $ is not defined
    at script:125:14
(anonymous) @ script:125
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 Stored JSON credentials: [REDACTED]
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 Begin ConnectionManager constructor
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 returning instance from getOrAddApiClient
main.jellyfin.bundle.js?22b44eb501cfeefd535d?deferred=true&ts=1735485786987:2 [jellyfin-web]
version: 10.10.3
commit: 
build: 10.10.3
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 returning instance from getOrAddApiClient
main.jellyfin.bundle.js?22b44eb501cfeefd535d?deferred=true&ts=1735485786987:2 loading installed plugins
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 returning instance from getOrAddApiClient
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 Begin connect
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 Begin getAvailableServers
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 Begin connectToServers, with 1 servers
node_modules.jellyfin-apiclient.bundle.js?22b44eb501cfeefd535d:2 begin connectToServer
```

`script` being Editor's Choice script, naturally.

If you have ADB set up on your phone (or the Android emulator), you can install a _debug_ build of the Jellyfin Android application (https://github.com/jellyfin/jellyfin-android/releases/latest) and open `chrome://inspect` in Chrome on your desktop to get access to the Android Jellyfin's app JavaScript console etc.

**Comment by lachlandcp** [comment]
Fixed in 91eb1b7. Thanks for the steer!

**Comment by qwerty12** [comment]
Thanks for the plugin 🫡
